JS Basics/Loops/Problem_4/index.js

```diff
@@ -0,0 +1,27 @@
+let allProps = [];
+let smallest = allProps[0], largest = allProps[0];
```

Comment:
At this stage allProps[0] is undefined.
No need to assign undefined to an undefined variable.

let smallest, largest;

is enough,